src/zenml/config/server_config.py

```diff
@@ -152,6 +153,9 @@ class ServerConfiguration(BaseModel):
             server.
         workload_manager_implementation_source: Source pointing to a class
             implementing the workload management interface.
+        max_concurrent_template_runs: The maximum number of concurrent template
+            runs that can be executed on the server. If not specified, the
+            default value of 1/4 of the thread pool size will be used.
```

Comment:
```suggestion
        max_concurrent_template_runs: The maximum number of concurrent template
            runs that can be executed on the server.
```

The last part is no longer accurate, is it ?